dhall-docs/README.md

```diff
@@ -1,6 +1,6 @@
 # `dhall-docs`
 
-:construction: **This tool is on development phase yet. It is not usable right now.**
+:construction: **This tool is on development phase yet. You can use it but you may expect breaking changes on each new release until the 1.0 release.**
```

Comment:
I would personally not worry keeping all breaking changes before the 1.0 release.  Any tool that has enough users will require a slow stream of slightly breaking changes over time to keep up with their evolving needs.

The rule of thumb I always use is that the users decide when a given tool is out of beta, not the author.  This is one of the reasons I always make the first published version of my packages 1.0.0, in recognition of the fact that users might put my tool into production regardless of whether or not I warn them not to.

Reply:
I understand your suggestion, thanks!

I think I'll leave the warning about the current status of the tool, it may be a good notice to people though:

```suggestion
:construction: **This tool is on development phase yet**
```